pkgs/development/tools/purescript/spago/default.nix

```diff
@@ -18,6 +18,7 @@ mkDerivation {
     rev = "a4679880402ead320f8be2f091b25d30e27b62df";
     fetchSubmodules = true;
   };
+  patches = [ ./spago-0.12.1-dhall-1.28.0.patch ];
```

Comment:
This file is generated with `cabal2nix`.

Could you instead add these patches to `pkgs/developement/haskell/configuration-common.nix`?

Reply:
Done.